[2023-09-04T06:02:14.169+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-16T00:00:00+00:00 [queued]>
[2023-09-04T06:02:14.182+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-16T00:00:00+00:00 [queued]>
[2023-09-04T06:02:14.183+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:14.202+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-03-16 00:00:00+00:00
[2023-09-04T06:02:14.207+0000] {standard_task_runner.py:57} INFO - Started process 5211 to run task
[2023-09-04T06:02:14.210+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-03-16T00:00:00+00:00', '--job-id', '389', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp8j_ugj6u']
[2023-09-04T06:02:14.213+0000] {standard_task_runner.py:85} INFO - Job 389: Subtask delete_data_from_table
[2023-09-04T06:02:14.271+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-16T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:14.376+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-16T00:00:00+00:00'
[2023-09-04T06:02:14.377+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-03-16' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:02:14.387+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:14.395+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:14.408+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-03-16' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:02:14.412+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:02:14.424+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220316T000000, start_date=20230904T060214, end_date=20230904T060214
[2023-09-04T06:02:14.463+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:14.484+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:04:59.247+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-16T00:00:00+00:00 [queued]>
[2023-09-04T06:04:59.273+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-16T00:00:00+00:00 [queued]>
[2023-09-04T06:04:59.274+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:04:59.320+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-03-16 00:00:00+00:00
[2023-09-04T06:04:59.331+0000] {standard_task_runner.py:57} INFO - Started process 6527 to run task
[2023-09-04T06:04:59.339+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-03-16T00:00:00+00:00', '--job-id', '936', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpaeai773i']
[2023-09-04T06:04:59.344+0000] {standard_task_runner.py:85} INFO - Job 936: Subtask delete_data_from_table
[2023-09-04T06:04:59.496+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-16T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:04:59.730+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-16T00:00:00+00:00'
[2023-09-04T06:04:59.733+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-03-16' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:04:59.754+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:59.772+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:59.796+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-03-16' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:04:59.806+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:04:59.839+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220316T000000, start_date=20230904T060459, end_date=20230904T060459
[2023-09-04T06:04:59.916+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:04:59.981+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
